packages/timescaledb.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Timescaledb < Pgpm::Package
```

Comment:
JFYI there is a TimescaleDB package (https://github.com/postgres-pm/pgpm/blob/master/packages/timescale/timescaledb.rb)